index.js

```diff
@@ -0,0 +1,5 @@
+import App from './modules/App.js';
+
+const app = new App();
+
+app();
```

Comment:
- Optional
You have `app()` which attempts to invoke the `app` as if it were a function. However, it is intended to be an instance of the App class. You might need to call a specific method on the app instance instead of invoking it directly. Currently it is throwing this error in the console. ⬇️
![image](https://github.com/tasnimchy96/bookStore-es6/assets/33268927/78a9f6cb-b3dc-40f6-808d-be4662ddff4a)